pkg/interface/src/views/apps/chat/components/ChatWindow.tsx

```diff
@@ -256,27 +269,12 @@ export default class ChatWindow extends Component<ChatWindowProps, ChatWindowSta
 
     const unreadMarkerRef = this.unreadMarkerRef;
 
-    const messages = new BigIntOrderedMap();
     let lastMessage = 0;
 
-    [...envelopes]
-      .sort((a, b) => a.number - b.number)
-      .forEach(message => {
-        const num = bigInt(message.number);
-        messages.set(num, message);
-        lastMessage = message.number;
-      });
-
-    stationPendingMessages
-      .sort((a, b) => a.when - b.when)
-      .forEach((message, index) => {
-        const idx = bigInt(index + 1); // To 1-index it
-        messages.set(bigInt(mailboxSize).add(idx), message);
-        lastMessage = mailboxSize + index;
-      });
-
     const messageProps = { association, group, contacts, hideAvatars, hideNicknames, remoteContentPolicy, unreadMarkerRef, history, api };
 
+    const keys = graph.keys().reverse();
```

Comment:
Why do you need to do this?

Reply:
so we can get the next and previous messages